content.tex

```diff
@@ -53,7 +56,7 @@ \section{Monitoring}
 A concerted effort will be made to be sure that one cohesive toolset is used for all teams who need to provide out of hour support for Rubin.
 A phone list will exist in order to reach/wake-up staff for In-Person summit out of hours support.  It will be made available to all observing staff, management and all staff members who are on the In-Person out of hours support list.  Everyone on this list must agree to be on the phone list a-priori.
 
-\section{Rubin vNOC - TBD}
+\section{Rubin Virtual Network Operations Center - TBD}
```

Comment:
why the TBD?

Reply:
The TBD was from Cristian's doc.  Not sure - would need to ask him to resolve it.